src/test/java/com/kodilla/ecommercee/domain/CartEntityTestSuite.java

```diff
@@ -108,35 +109,31 @@ public void testDeleteCart() {
     @Test
     public void testRelationWithUser() {
         //given
-        Cart cart = new Cart();
-        cartDao.save(cart);
-        User user = new User("username3", 1, "hgdasa");
+        User user = new User("username3", 1, "hgdasa", LocalDateTime.now());
+        Cart cart = new Cart(user);
```

Comment:
ja ustawiam to setterem, ja kuzyejsz tego w tej kolejności to nie będziesz mógł utworzyć cart po user, bo user jeszcze nie jest zapisane. znów zwiazane jest to z CascadeType.ALL które usuwamy